src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -250,15 +259,12 @@ class MoreInfoLight extends LitElement {
         align-items: center;
       }
 
-      .content.is-on {
-        margin-top: -16px;
-      }
-
       .content > * {
         width: 100%;
         max-height: 84px;
         overflow: hidden;
         padding-top: 16px;
+        margin-top: -16px;
```

Comment:
why a padding op 16 and negative margin of -16? Can't we remove the padding if you want it higher?

Reply:
True that 👍